server/routes/acctRoutes.js

```diff
@@ -4,12 +4,12 @@ import auth from '../authorization/auth';
 
 const AcctRouter=Router();
 
-AcctRouter.get('/accounts',auth.authorization,accounts.getAcctInfo);
-AcctRouter.post('/accounts',auth.authorization,accounts.createAccount);
-AcctRouter.patch('/account/:accountNumber',auth.authorization,accounts.updateAccount);
-AcctRouter.delete('/accounts/:accountNumber',auth.authorization,accounts.deleteAccount);
+AcctRouter.get('/v1/accounts',auth.authorization,accounts.getAcctInfo);
```

Comment:
Are all these v1s part of challenge 2 work?

Reply:
yes